File: java/src/main/java/ai/buble/sdk/generations/GenerationTaskError.java
package ai.buble.sdk.generations;

/**
 * Task-level generation error.
 */
public class GenerationTaskError {
    private String code;
    private String message;

    public String getCode() { return code; }
    public void setCode(String code) { this.code = code; }
    public String getMessage() { return message; }
    public void setMessage(String message) { this.message = message; }
}

File: dotnet/src/Buble.Sdk/Generations/GenerationTaskError.cs
using System.Text.Json;
using System.Text.Json.Serialization;

namespace Buble.Sdk.Generations;

public sealed class GenerationTaskError
{
    [JsonPropertyName("code")]
    public string? Code { get; set; }

    [JsonPropertyName("message")]
    public string? Message { get; set; }

    [JsonExtensionData]
    public Dictionary<string, JsonElement>? AdditionalProperties { get; set; }
}
